app-crypt/heimdal/heimdal-7.1.0.ebuild

```diff
@@ -15,9 +15,13 @@ SRC_URI="https://github.com/${PN}/${PN}/releases/download/${P}/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd"
-IUSE="afs +berkdb caps hdb-ldap ipv6 otp +pkinit selinux ssl static-libs test X"
+IUSE="+berkdb +pkinit X afs caps hdb-ldap ipv6 libressl otp selinux ssl static-libs test"
```

Comment:
I think that sort means "afs +berkdb caps hdb-ldap ipv6 libressl  otp +pkinit selinux ssl static-libs test X" is the way , it is way too confusing, aha.